modeller/chapter.md

```diff
@@ -155,13 +161,14 @@ To draw an item to the screen, we need to convert between a few different coordi
 ![Transformation Pipeline](newtranspipe.png?raw=true)
```

Comment:
This diagram needs more introduction and explanation. Can you provide a quick explanation for each of the "spaces", or at least all the ones you refer to in the rest of the chapter? The diagram also uses words (mesh, frustum) which are not defined or used in the rest of the chapter - you should acknowledge that in your explanation. Maybe something like, "This diagram includes more information than we discuss in this chapter; disregard the transitions from ... to ..."  Alternately, maybe you could edit the diagram so it only includes the relevant information?